.github/workflows/build.yml

```diff
@@ -25,47 +25,17 @@ on:
       - 'v*.*.*'
 
 jobs:
-  macos-latest:
-    name: macos-latest
-    runs-on: macos-latest
-    steps:
-      - uses: actions/checkout@v3
-      - name: 'Run: Compile, Test, Pack'
-        run: ./build.cmd Compile Test Pack
-      - name: 'Publish: test-results'
-        uses: actions/upload-artifact@v3
-        with:
-          name: test-results
-          path: artifacts/test-results
-      - name: 'Publish: packages'
-        uses: actions/upload-artifact@v3
-        with:
-          name: packages
-          path: artifacts/packages
   ubuntu-latest:
     name: ubuntu-latest
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - name: 'Run: Compile, Test, Pack'
-        run: ./build.cmd Compile Test Pack
-      - name: 'Publish: test-results'
-        uses: actions/upload-artifact@v3
-        with:
-          name: test-results
-          path: artifacts/test-results
-      - name: 'Publish: packages'
-        uses: actions/upload-artifact@v3
-        with:
-          name: packages
-          path: artifacts/packages
-  windows-latest:
-    name: windows-latest
-    runs-on: windows-latest
-    steps:
-      - uses: actions/checkout@v3
-      - name: 'Run: Compile, Test, Pack'
-        run: ./build.cmd Compile Test Pack
```

Comment:
Not a blocker at all, just wondering:

Would it be a hard task to set up the master build workflow (either here or in NUKE) to run the Compile, Test, Pack actions for all platforms, then run Publish only when all platforms build successfully?

It would be nice to make completely sure that all platforms are ok before releasing a package. (In my experience, things can occasionally get messed up even after PR checks are successful.)

Reply:
I'm thinking of a setup like this:

![image](https://github.com/adams85/acornima/assets/31276480/80b23c3a-e28a-4902-b8d3-8a9f5c15b37f)

(Except for the test coverage check, which we don't have currently.)

Reply:
Unfortunately the YAML generation isn't that flexible at the moment to expression this, but probably could be extended to output such flows. I usually just check that main/master is green and then push the tag against green commit.

There's always of course possibility to write the YAML by hand based on  generated one too. NUKE tries to create the flow that works across different CI systems.

I think code coverage comes almost out of the box too with ITest.

Reply:
> I usually just check that main/master is green and then push the tag against green commit.

If my understanding is correct, this PR makes a change so that master is built on Ubuntu only. So the master being green won't tell much about the two other platforms, will it?

> There's always of course possibility to write the YAML by hand based on generated one too. 

Alright, it's good enough what we have now, let's save this improvement for a future PR.

> I think code coverage comes almost out of the box too with ITest.

Sounds promising!